### PR TITLE
Hotfix/sf star update

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

With the patch release of [dbt-utils v0.7.3](https://github.com/dbt-labs/dbt-utils/releases/tag/0.7.3) the snowflake uppercase fix no longer is needed and thus is removed from relevant models.

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide explanation how the change is non breaking below.)

This simply removes a Snowflake conditional that is no longer needed and is actually breaking users runs currently.

**Is this PR in response to a previously created Issue**

- [x] Yes https://github.com/fivetran/dbt_iterable/issues/9
- [ ] No 

**How did you test the PR changes?** 

 
- [x] CircleCi  
- [ ] Other (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 
- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🧯 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: integration_tests/dbt_project.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 config-version: 2
 
 name: 'iterable_integration_tests'
-version: '0.2.0'
+version: '0.2.1'
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:4]: Small fix to keep the two versions up to date for consistency
```suggestion
version: '0.3.1'
```

File: models/iterable__campaigns.sql [hunk 2]
```diff
@@ -35,7 +35,7 @@ with campaign_event_metrics as (
 ), campaign_join as (
 
     {% set exclude_fields = [ 'campaign_id', 'template_id'] %} -- these are both in campaigns
-    {% set exclude_fields = exclude_fields | upper if target.type == 'snowflake' else exclude_fields %} -- snowflake needs uppercase :)
+    {% set exclude_fields = exclude_fields %}
```
Comment (fivetran-joemarkiewicz) [inline models/iterable__campaigns.sql:38]: We can ideally remove this line, correct? This `set` statement is doing the same as the above line.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
Looks good!

I have one small question regarding if we truly need to keep the `set` function in one space. I believe we should be able to simply remove this. Let me know your thoughts and then you can have me re-review in case there are any other changes based off my comment.

Thanks!

File: models/iterable__campaigns.sql (hunk 2 above)
Comment (fivetran-reneeli) [reply models/iterable__campaigns.sql]: You're right, I didn't even realize it became redundant!

**Comment by fivetran-reneeli** [comment]
> Looks good!
> 
> I have one small question regarding if we truly need to keep the `set` function in one space. I believe we should be able to simply remove this. Let me know your thoughts and then you can have me re-review in case there are any other changes based off my comment.
> 
> Thanks!

Thanks for pointing that out. I updated the PR; check has passed if you'd like to re-review!

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM! 🕺

**Pull request merged**